packages/core/src/components/Table/TableCell/TableCell.tsx

```diff
@@ -297,78 +127,111 @@ export const HvTableCell = forwardRef<HTMLElement, HvTableCellProps>(
 
     const TableCell = useMemo(() => StyledTableCell(Component), [Component]);
 
+    const sortedColorValue = getVarValue(theme.table.rowSortedColor);
```

Comment:
I believe we are having the same problem here as we had with the stripped table in the `TableRow` component right?

Reply:
The fix for that, was not working as intended, the first render was not correct, I sort of reverted it in the next commit and suggested  @zettca that maybe is better to open a separate issue to find a better way to deal with that problem

Reply:
Okay, makes sense!